src/objects/zcl_abapgit_object_bdef.clas.abap

```diff
@@ -204,72 +274,101 @@ CLASS zcl_abapgit_object_bdef IMPLEMENTATION.
-
-        <lv_source> = mo_files->read_string( 'asbdef' ).
-
-        tadir_insert( iv_package ).
 
+        lo_object_data = get_object_data( io_xml ).
         lv_transport_request = get_transport_req_if_needed( iv_package ).
 
-        IF zif_abapgit_object~exists( ) = abap_true.
-
-          " We need to populate created_at, created_by, because otherwise update  is not possible
-          fill_metadata_from_db( CHANGING cs_behaviour_definition = <ls_behaviour_definition> ).
-          li_object_data_model->set_data( <ls_behaviour_definition> ).
+        CREATE DATA lr_wbobjtype TYPE ('WBOBJTYPE').
+        ASSIGN lr_wbobjtype->* TO <ls_wbobjtype>.
+        ASSIGN COMPONENT 'OBJTYPE_TR' OF STRUCTURE <ls_wbobjtype> TO <lv_field>.
+        <lv_field> = 'BDEF'.
+        ASSIGN COMPONENT 'SUBTYPE_WB' OF STRUCTURE <ls_wbobjtype> TO <lv_field>.
+        <lv_field> = 'BDO'.
 
-          CALL METHOD li_wb_object_operator->('IF_WB_OBJECT_OPERATOR~UPDATE')
-            EXPORTING
-              io_object_data    = li_object_data_model
-              transport_request = lv_transport_request.
+        CREATE DATA lr_category TYPE ('WBADT_RESOURCE_CATEGORY').
+        ASSIGN lr_category->* TO <lv_category>.
 
-        ELSE.
+        CALL METHOD ('CL_BLUE_WB_UTILITY')=>('GET_RESOURCE_CATEGORY')
+          EXPORTING
+            is_object_type = <ls_wbobjtype>
+          RECEIVING
+            result         = <lv_category>.
 
-          li_object_data_model->set_data( <ls_behaviour_definition> ).
+        lo_wb_object_operator = get_wb_object_operator( ).
 
-          CALL METHOD li_wb_object_operator->('IF_WB_OBJECT_OPERATOR~CREATE')
-            EXPORTING
-              io_object_data    = li_object_data_model
-              data_selection    = 'P' " if_wb_object_data_selection_co=>c_properties
-              package           = iv_package
-              transport_request = lv_transport_request.
+        tadir_insert( iv_package ).
 
-          CALL METHOD li_wb_object_operator->('IF_WB_OBJECT_OPERATOR~UPDATE')
-            EXPORTING
-              io_object_data    = li_object_data_model
-              data_selection    = 'D' " if_wb_object_data_selection_co=>c_data_content
-              transport_request = lv_transport_request.
 
+        IF zif_abapgit_object~exists( ) = abap_false.
+          CASE <lv_category>.
+            WHEN '1'. "if_wb_adt_plugin_resource_co=>co_sfs_res_category_atomic.
+              CALL METHOD lo_wb_object_operator->('IF_WB_OBJECT_OPERATOR~CREATE')
+                EXPORTING
+                  io_object_data    = lo_object_data
+                  data_selection    = 'AL' " if_wb_object_data_selection_co=>c_all_data
+                  version           = 'I'
+                  package           = iv_package
+                  transport_request = lv_transport_request.
+            WHEN '2'. "if_wb_adt_plugin_resource_co=>co_sfs_res_category_compound_s.
+              CALL METHOD lo_wb_object_operator->('CREATE')
```

Comment:
Doesn't work for me on S/4HANA 1909 SP02 

![image](https://user-images.githubusercontent.com/17437789/95880266-9c54bb00-0d77-11eb-9ba1-df9784a06e93.png)

CREATE method doesn't exist in CL_WB_OBJECT_OPERATOR.
![image](https://user-images.githubusercontent.com/17437789/95880363-b9898980-0d77-11eb-9a92-69978ccd306c.png)